src/index.c

```diff
@@ -1092,7 +1092,7 @@ int NI_SkipTo_O(void *ctx, t_docId docId, RSIndexResult **hit) {
     // This doc-id was deleted
     return INDEXREAD_NOTFOUND;
   }
-  RS_LOG_ASSERT_FMT(nc->lastDocId == docId, "Expected docId to be %llu, got %llu", docId, nc->lastDocId);
+  RS_LOG_ASSERT_FMT(nc->lastDocId == docId, "Expected docId to be %zu, got %zu", docId, nc->lastDocId);
```

Comment:
Why change format from `%llu` to `%zu` - the type of `t_docId` is `uint64_t` not `size_t`